.gitignore

```diff
@@ -114,3 +114,4 @@ dist
 
 # IDE files
 .idea
+*.iml
```

Comment:
Normally I'd say, this isn't relevant to the PR, so make is a separate one, but I'll let it slide this time :)